app/views/slides/show.html.erb

```diff
@@ -47,12 +47,12 @@
     <% end %>
 
     <br />
-    <label>Shown on <%= pluralize(@slide.signs.length, 'sign')%>:</label>
+    <label>Shown on <%= pluralize(accessible_sign_slides(@slide).length, 'sign')%>:</label>
     <ul class="signs-list">
-      <% @slide.sign_slides.each do |ss| %>
+      <% accessible_sign_slides(@slide).each do |ss| %>
```

Comment:
I'm pretty sure that won't happen. In the case where there are no sign_slides accessible to the current user because the associated sign(s) are private, it will return an empty array.